assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -70,6 +70,15 @@ production: &base
     email_display_name: {{GITLAB_EMAIL_DISPLAY_NAME}}
     email_reply_to: {{GITLAB_EMAIL_REPLY_TO}}
     email_subject_suffix: '{{GITLAB_EMAIL_SUBJECT_SUFFIX}}'
+    email_smime:
```

Comment:
I think we need also a logic here to remove those values if `GITLAB_EMAIL_SMIME_ENABLE` is false. Check for the logic https://github.com/sameersbn/docker-gitlab/blob/master/assets/runtime/functions#L765 and https://github.com/sameersbn/docker-gitlab/blob/master/assets/runtime/config/gitlabhq/gitlab.yml#L633 for the yaml setup. This will remove the lines if it's not set and should the default behaviour.

Reply:
Added `email-smime` block. The s/mime email configurations is now removed if `GITLAB_EMAIL_SMIME_ENABLE` is false